package.json

```
@@ -101,15 +101,8 @@
       "repo": "desktop-client"
     }
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/pomerium/desktop-client.git"
-  },
-  "author": {
-    "name": "Pomerium Inc",
-    "email": "info@pomerium.com",
-    "url": "https://pomerium.io/"
-  },
+  "repository": "git+https://github.com/pomerium/desktop-client.git",
```

```suggestion
  "repository": "github:pomerium/desktop-client",
```

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository - Seems like we should stick with the previous format or try one of these short versions.  Thoughts?

---

switched it back